core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -241,9 +250,10 @@ private[group] class GroupMetadata(val groupId: String, initialState: GroupState
 
   def currentState = state
 
+  // TODO: deprecate the kick out logic for normal member's rejoin failure within rebalance timeout.
```

Comment:
In my opinion, it's better to just create a JIRA rather than a TODO comment, since those generally don't get addressed (e.g the one you removed with this PR) and don't contain as much context

Reply:
I plan to address this in the next PR (KIP-345). In case I'm not available for future work, others could easily pick up when seeing the comments :)

Reply:
Thanks for the info. Let me remove this comment.